eth/handler_eth.go

```diff
@@ -160,6 +184,15 @@ func (h *ethHandler) handleHeaders(peer *eth.Peer, headers []*types.Header) erro
 	return nil
 }
 
+func (h *ethHandler) purgeWhitelistMap(endBlockNum uint64) error {
```

Comment:
We can also add `bor_purgeWhitelistCheckpoint` RPC to remove checkpoint whitelisting and `bor_whitelistCheckpoint` to fetch all whitelisted checkpoints for read/debug purposes.